src/emulator/controller.ts

```diff
@@ -239,19 +246,29 @@ export async function startAll(options: any, noGui: boolean = false): Promise<vo
     }
 
     const rulesLocalPath = options.config.get("firestore.rules");
+    const foundRulesFile = rulesLocalPath && fs.existsSync(rulesLocalPath);
     if (rulesLocalPath) {
       const rules: string = path.join(options.projectRoot, rulesLocalPath);
       if (fs.existsSync(rules)) {
         args.rules = rules;
       } else {
-        utils.logWarning(
-          `Firestore rules file ${clc.bold(
-            rules
-          )} specified in firebase.json does not exist, starting Firestore emulator without rules.`
+        utils.logLabeledWarning(
+          "firestore",
+          `Firestore rules file ${clc.bold(rules)} specified in firebase.json does not exist.`
```

Comment:
Do we want to apply any kind of formatting to firebase.json (throughout)?

Reply:
You mean bold or something like that?

Reply:
In some messages, it looks like there are double quotes.  In other messages, there's no formatting or anything at all.

Just wondering if we could have consistent formatting throughout.  Not bold, but maybe just single quotes or backticks or something?

Totally not blocking, but while we're in here, thought we'd clean it up.